readme.md

```diff
@@ -125,9 +125,9 @@ Query string object that will be added to the request URL. This will override th
 
 Type: `number`, `object`
 
-Milliseconds to wait for a server to send response headers before aborting request with `ETIMEDOUT` error.
+Milliseconds to wait for the server to end the response before aborting request with `ETIMEDOUT` error.
 
-Option accepts `object` with separate `connect` and `socket` fields for connection and socket inactivity timeouts.
+Option accepts `object` with separate `connect`, `socket`, and `request` fields for connection, socket, and entire request timeouts.
```

Comment:
I know it's not from this change, but `Option accepts `object` with separate` => `This also accepts an `object` with separate`. And `fields` => `properties`